mlflow/pyfunc/scoring_server/__init__.py

```diff
@@ -132,6 +137,74 @@ def infer_and_parse_json_input(json_input, schema: Schema = None):
         )
 
 
+def _decode_json_input(json_input):
+    """
+    :param json_input: A JSON-formatted string representation of TF serving input or a Pandas
+                       DataFrame, or a stream containing such a string representation.
+    :return: A dictionary representation of the JSON input.
+    """
+    if isinstance(json_input, dict):
+        decoded_input = json_input
```

Comment:
We can just return here, right?
```suggestion
        return json_input
```